app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_likes_counter
+  def update_likes_counter
+    post.update(likes_count: post.likes.count)
```

Comment:
- [ ] This should be `likes_counter`, not `likes_count` based on your schema.

  ![image](https://github.com/Surafels/Blog-app/assets/54848363/202664e0-d312-4ad7-b772-309906e14089)